arm-analysisservices/2016-05-16/swagger/analysisservices.json

```diff
@@ -410,6 +410,11 @@
       "description": "Properties of Analysis Services resource.",
       "type": "object",
       "properties": {
+        "state": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The current state of Analysis Services resource. It allows more custom terminal state like Paused."
```

Comment:
@athipp - Did you verify with ARM team, whether you can add more values to provisioningState. My understanding is that ARM does not stop an RP to add more values. If that is true then don't you think using one property would be easier. It will be less confusing for customers.
@ravbhatnagar - Can you assert my understanding about ARM in the above statement?